pkg/storaged/stratis-details.jsx

```diff
@@ -57,6 +58,17 @@ const _ = cockpit.gettext;
 
 const fsys_min_size = 512 * 1024 * 1024;
 
+export function check_stratis_warnings(client, enter_warning) {
+    if (!client.features.stratis_grow_blockdevs)
+        return;
+
+    for (const p in client.stratis_pools) {
+        const blockdevs = client.stratis_pool_blockdevs[p] || [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230825-132955-2df3b918-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)